src/js/SAFE-config.js

```diff
@@ -0,0 +1,36 @@
+// Note that we import these values into "web_app_config" (so we can search for it)
```

Comment:
Hi Steve, since we aren't trying to run build tests on our WeVoteReactNative repository, I wonder if we can move src/js/config.js into .gitignore, and create a src/js/config-template.js that never contains private keys? (I will create an issue.)

Reply:
Removed